src/DotVVM.Compiler/DotVVM.Compiler.csproj

```diff
@@ -10,7 +10,7 @@ for some reason. What fun. -->
 
   <!-- Related to compilation -->
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;net461;net5.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;net472;net461;net5.0</TargetFrameworks>
```

Comment:
Did you forgot to remove net461? If not, you'll have to include it in the condition bellow